### DISCUSSION TITLE
Welcome to ProfilAnalyser Discussions!

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/ProfilAnalyser#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by see3d** [comment]
I have been banging my head against a wall for days, trying to find an easy way to compare two profiles in Cura 5.8.  I finally came across your YouTube video for this plugin.  This is the perfect functionality I was looking for.  The only problem is that it is not in the Cura Marketplace.  Are there plans to update it to work with the current version of Cura?

Edit:  I found an alternative method.  Open two instances of Cura and compare settings that way.  It's not as nice as yours, but it's workable.

**Comment by rufus-se** [comment]
Just saw this plugin in a YT video. Went straight to market place but it was'nt there. 
Is this an abandoned project or are there plans on upgrading to current versions?

**Comment by 5axes** [comment]
Normally the Plugin still works, but I've abandoned Cura for another slicer that's more active and modern in the features it offers.  So, not having the need to continue using these plugins nor the will to maintain them, we can consider the project as "Abandoned".
